### PR TITLE
Add cocoapods MO

### PR DESCRIPTION
**Pull request by yleflour**

### CODE REVIEW

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/add-cocoapods/react-native/setup/add-cocoapods.mo.md">add-cocoapods.mo.md</a>**: Seems you do not need a *Troubleshoot* part
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: react-native/setup/add-cocoapods.mo.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## Steps:
+
+### 1. Add cocoapods (~5min)
+
+- In your *Gemfile* add the line `cocoapods`
+- Run `bundle install`
+- Go into the iOS project folder (`cd ios`)
+  - Initialize pods with `bundle exec pod init`
+  - Add `pods` to *.gitignore*
```
Comment (tychota) [inline react-native/setup/add-cocoapods.mo.md:18]: That is not a good ios practice. They version pods. No joke.

File: react-native/setup/add-cocoapods.mo.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+> **Checks:**
+> - You now have a *projectName.xcworkspace* file in the `ios` folder
+> - You can run your project by opening *projectName.xcworkspace* with XCode or `react-native run-ios`
+> The *pods* folder shouldn't show up in the git diffs
+
+### 2. Add your first Pod (~2min)
+
+- Open the `Podfile`
+  - Add `pod '<POD_NAME>', '~> <Version>'` inside the target
```
Comment (tychota) [inline react-native/setup/add-cocoapods.mo.md:32]: Precise example if possible.

**Review by tychota — APPROVED** [review]

**Comment by tychota** [comment]
Troubleshooting

Pod version,
Dynamic framework
Ios version

File: react-native/setup/add-cocoapods.mo.md (hunk 1 above)
Comment (tychota) [reply react-native/setup/add-cocoapods.mo.md:18]: https://stackoverflow.com/a/22323702

You actually have the choice but it is safer to version it.

Comment (yleflour) [reply react-native/setup/add-cocoapods.mo.md:18]: This is shameful, not doing that until we face an issue

**Pull request merged**